test/framework/cluster.go

```diff
@@ -583,29 +586,59 @@ func (e *ClusterE2ETest) WithClusterConfig(fillers ...api.ClusterConfigFiller) *
 	return e
 }
 
-func (e *ClusterE2ETest) ImportImages(opts ...CommandOpt) {
-	importImagesArgs := []string{"import-images", "-f", e.ClusterConfigLocation}
-	e.RunEKSA(importImagesArgs, opts...)
-}
-
+// DownloadArtifacts runs the EKS-A `download artifacts` command with appropriate args.
 func (e *ClusterE2ETest) DownloadArtifacts(opts ...CommandOpt) {
 	downloadArtifactsArgs := []string{"download", "artifacts", "-f", e.ClusterConfigLocation}
+	if getBundlesOverride() == "true" {
+		downloadArtifactsArgs = append(downloadArtifactsArgs, "--bundles-override", defaultBundleReleaseManifestFile)
+	}
 	e.RunEKSA(downloadArtifactsArgs, opts...)
-	if _, err := os.Stat("eks-anywhere-downloads.tar.gz"); err != nil {
+	if _, err := os.Stat(defaultDownloadArtifactsOutputLocation); err != nil {
 		e.T.Fatal(err)
 	} else {
-		e.T.Log("Downloaded artifacts saved at eks-anywhere-downloads.tar.gz")
+		e.T.Logf("Downloaded artifacts tarball saved at %s", defaultDownloadArtifactsOutputLocation)
 	}
 }
 
-// ExtractDownloadedArtifacts extract the downloaded artifacts.
+// ExtractDownloadedArtifacts extracts the downloaded artifacts.
 func (e *ClusterE2ETest) ExtractDownloadedArtifacts(opts ...CommandOpt) {
-	if _, err := os.Stat("eks-anywhere-downloads.tar.gz"); err != nil {
+	e.T.Log("Extracting downloaded artifacts")
+	e.Run("tar", "-xf", defaultDownloadArtifactsOutputLocation)
+}
+
+// DownloadImages runs the EKS-A `download images` command with appropriate args.
+func (e *ClusterE2ETest) DownloadImages(opts ...CommandOpt) {
+	downloadImagesArgs := []string{"download", "images", "-o", defaultDownloadImagesOutputLocation}
+	if getBundlesOverride() == "true" {
+		downloadImagesArgs = append(downloadImagesArgs, "--bundles-override", defaultBundleReleaseManifestFile)
```

Comment:
download images should take into account of the bundle released downloaded from "download artifact" command as well. There is opportunity to reuse the code.

Reply:
This is currently reusing the bundles override code that the create/upgrade/delete workflows are using for tests. It's going to be the same bundle in either case

Reply:
It's not "necessary" to change if either case the same, but, they are not the same (either due to some bug or by design). Manifesturl in artifact bundlerelease.yaml points to a local file, while the bundle-release downloaded through curl points to a remote URL.

Reply:
Oh I just meant, as far as images goes it should ideally preserve the same URIs right? only the S3 URI will be replaced by local filepaths

Reply:
I am not opposed to changing if it makes it more proper, just an observation

Reply:
Done!